src/battle_message.c

```diff
@@ -3918,14 +3934,6 @@ static u32 GetEnemyMonCount(u32 firstId, u32 lastId, bool32 onlyAlive)
     return count;
 }
 
-static bool32 IsBattlerHpLow(u32 battler)
-{
-    if ((gBattleMons[battler].hp * 100) / gBattleMons[battler].maxHP < 25)
-        return TRUE;
-    else
-        return FALSE;
-}
-
 u32 ShouldDoTrainerSlide(u32 battlerId, u32 which)
 {
```

Comment:
can we clean up some of the code below by calling frequently-used functions e.g. `GetEnemyMonCount(firstId, lastId, TRUE)` , or initializing variables like currHp, maxHp for the hp threshold checks?

Reply:
`GetEnemyMonCount` is already called.

As for the thresholds, I suppose I could, yeah. I'll see what I can come up with.

Reply:
Check up my ping in the Discord server later. I need your opinion on a function that I wrote in an attempt to make it those maxHp checks more readable.